src/segment/segmentTemplate.js

```diff
@@ -147,7 +147,8 @@ export const segmentsFromTemplate = (attributes, segmentTimeline) => {
   const mapSegment = urlTypeToSegment({
     baseUrl: attributes.baseUrl,
     source: constructTemplateUrl(initialization.sourceURL, templateValues),
-    range: initialization.range
+    range: initialization.range,
+    customRedirectUrl: attributes.customRedirectUrl
```

Comment:
I would have simply name it for what it is, subtitleConverterUri.
Other than that, looks ok